spec/javascripts/components/story/expanded_story/expanded_story_description/expanded_story_description_spec.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { shallow, mount } from 'enzyme';
+import ExpandedStoryDescription from 'components/story/ExpandedStory/ExpandedStoryDescription/index';
+
+describe('<ExpandedStoryDescription />', () => {
+  const defaultProps = () => ({
+    story: {},
+    onEdit: sinon.spy(),
+    disabled: false
+  });
+
+  it('renders component title', () => {
+    const story = { description: null, _editing: { description: null } };
+
+    const wrapper = mount(
```

Comment:
Does it need to be `mount`? If so, we don't need to import `shallow` in this file.